packages/form/test/TextArea.astro.test.js

```diff
@@ -0,0 +1,110 @@
+import { expect } from 'chai';
+import { describe, beforeEach, it } from 'mocha';
+import { getComponentOutput } from 'astro-component-tester';
+import { cleanString } from './utils/index.js';
+
+describe('TextArea.astro test', () => {
```

Comment:
Awesome, thanks for adding the tests!! 🙏

Reply:
No problems! ❤️